2-structured-data/src/DataModel/Sql.php

```diff
@@ -42,7 +42,7 @@ public function __construct($dsn, $user, $password)
             'id serial PRIMARY KEY ',
             'title VARCHAR(255)',
             'author VARCHAR(255)',
-            'publishedDate VARCHAR(255)',
+            'publisheddate VARCHAR(255)',
```

Comment:
Why are we lowercasing this? `imageUrl` below has the same casing, as does `createdBy`. It'd be best to be consistent unless we have a strong reason to change it. If we need to change it, we should change all of them to snake case (i.e. `published_date`